--- V0001__base.sql	2023-01-22 13:32:03
+++ tmp/V0001__base.sql	2022-11-17 22:29:51
@@ -20,14 +20,14 @@
 -- Name: adminpack; Type: EXTENSION; Schema: -; Owner: -
 --
 
-CREATE EXTENSION IF NOT EXISTS adminpack WITH SCHEMA pg_catalog;
+--CREATE EXTENSION IF NOT EXISTS adminpack WITH SCHEMA pg_catalog;
 
 
 --
 -- Name: EXTENSION adminpack; Type: COMMENT; Schema: -; Owner: 
 --
 
-COMMENT ON EXTENSION adminpack IS 'administrative functions for PostgreSQL';
+--COMMENT ON EXTENSION adminpack IS 'administrative functions for PostgreSQL';
 
 
 --
